threshsign/src/CMakeLists.txt

```diff
@@ -8,7 +8,12 @@ set(common_source_files
     IThresholdVerifier.cpp
     VectorOfShares.cpp
     ThresholdSignaturesTypes.cpp
-)
+    # TODO: create a separate target
+    eddsa/EdDSAMultisigFactory.cpp
```

Comment:
These files are in the eddsa directory, and should be namespaced correspondingly.
No need to duplicate it in the filename, i.e. `eddsa/MultisigFactory.cpp`